test/generic.spec.js

```diff
@@ -0,0 +1,174 @@
+var fs = require('fs');
+var Promise = require('bluebird');
+var readFile = Promise.promisify(require('fs').readFile);
+var expect = require('chai').expect;
+var async = require('async');
+var Converter = require('csvtojson').Converter;
```

Comment:
To answer your question - yes It is the same for me that I can install it through npm but it is not included in dependencies in package.json and since I don't have it installed globally I can't use it.